srcCxx/ShapeType_main.cxx.orig

```diff
@@ -0,0 +1,199 @@
+/* 
```

Comment:
I think this file should not be added to the git repository.

Reply:
Removed.  Should we add a .gitignore for .o files too ?